wordpress-mobile/gutenberg-mobile
# Expand supported endpoints with WP hook

Related to https://github.com/WordPress/gutenberg/pull/49371#discussion_r1151755664.

**Related PR:**
* https://github.com/WordPress/gutenberg/pull/49581

**To test:**
1. Apply the following patch file:
```patch
diff --git forkSrcPrefix/packages/react-native-editor/src/api-fetch-setup.js forkDstPrefix/packages/react-native-editor/src/api-fetch-setup.js
index 6d2f401e04e26581a04b5145fb382f582f578b08..3c8dee9d0158228cd593966bd0d18b31c15b7679 100644
--- forkSrcPrefix/packages/react-native-editor/src/api-fetch-setup.js
+++ forkDstPrefix/packages/react-native-editor/src/api-fetch-setup.js
@@ -76,6 +76,7 @@ export const isPathSupported = ( path, method ) => {
 		'native.supported_endpoints',
 		SUPPORTED_ENDPOINTS
 	);
+	console.log( { supportedEndpoints } );
 	return supportedEndpoints[ method ].some( ( pattern ) =>
 		pattern.test( path )
 	);
```
2. Open the app with the local Metro server.
3. Observe the console logs and check that the new endpoint added (`/wpcom\/v2\/(media)\/.*/i`) in this PR is printed.

PR submission checklist:
